encoders/video/VideoPaddleEncoder/__init__.py

```diff
@@ -0,0 +1,80 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import numpy as np
+
+from jina.executors.decorators import batching, as_ndarray
+from jina.executors.encoders.frameworks import BasePaddleEncoder
+
+
+class VideoPaddleEncoder(BasePaddleEncoder):
+    """
+    :class:`VideoPaddleEncoder` encodes data from a ndarray, potentially B x T x (Channel x Height x Width) into a
+        ndarray of `B x D`.
+    Internally, :class:`VideoPaddleEncoder` wraps the models from `paddlehub`.
+    https://github.com/PaddlePaddle/PaddleHub
+    """
+
+    def __init__(self,
+                 model_name: str = None,
+                 output_feature: str = None,
+                 pool_strategy: str = None,
+                 channel_axis: int = 2,
+                 *args, **kwargs):
+        """
+
+        :param model_name: the name of the model. Supported models include ``tsn_kinetics400``, ``stnet_kinetics400``,
+            ``tsm_kinetics400``
+        :param output_feature: the name of the layer for feature extraction. Please use the following values for the
+            supported models:
+            ``tsn_kinetics400``: `@HUB_tsn_kinetics400@reduce_mean_0.tmp_0`
+            ``stnet_kinetics400``: ``@HUB_stnet_kinetics400@reshape2_6.tmp_0``
+            ``tsm_kinetics400``: ``@HUB_tsm_kinetics400@reduce_mean_0.tmp_0``
+        :param pool_strategy: the pooling strategy
+            - `None` means that the output of the model will be the output feature.
+            - `mean` means that global average pooling will be applied to the output feature, and thus the output of the
+                model will be a 2D tensor.
+            - `max` means that global max pooling will be applied.
+        """
+        super().__init__(*args, **kwargs)
+        self.model_name = model_name or 'tsn_kinetics400'
```

Comment:
instead of putting this `or`, let's put these values as default init ones?

Reply:
Yeah, sure, good idea.